src/ds/d500/d500-color.cpp

```diff
@@ -82,6 +84,14 @@ namespace librealsense
         _color_device_idx = add_sensor(color_ep);
     }
 
+    void d500_color::register_features()
+    {
+        register_feature( auto_exposure_roi_feature::ID,
+                          std::make_shared< auto_exposure_roi_feature >( get_color_sensor(), _hw_monitor, true ) );
+
+        d500_device::register_features();
```

Comment:
Not needed -- this function should not be virtual (please change its name to be `register_color_features`) and is called from d500_device::init().

Reply:
Removed `virtual` property from `d500_device::register_features` and renamed here.